server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -100,17 +100,10 @@ export class ScriptoriumLambda implements IPartitionLambda {
             ...message,
             mongoTimestamp: new Date(message.operation.timestamp),
         }));
-        return this.opCollection
```

Comment:
It looks like this was returning the result of the insertion call. Should we retain that functionality?

Reply:
Good catch! Yes you are right